src/pretalx/orga/templates/orga/submission/content.html

```diff
@@ -97,6 +97,7 @@
         {% if form.abstract %}{% bootstrap_field form.abstract layout='event' %}{% endif %}
         {% if form.description %}{% bootstrap_field form.description layout='event' %}{% endif %}
         {% if form.notes %}{% bootstrap_field form.notes layout='event' %}{% endif %}
+        {% if form.orga_notes %}{% bootstrap_field form.orga_notes layout='event' %}{% endif %}
```

Comment:
Why wouldn't the field be present? Do you want it to be hidden towards reviewers?